pkg/coscheduling/coscheduling.go

```diff
@@ -137,6 +138,30 @@ func (cs *Coscheduling) PreFilter(ctx context.Context, state *framework.CycleSta
 	return framework.NewStatus(framework.Success, "")
 }
 
+// PostFilter is use for rejecting the group pods if a pod can not pass resource check when filter.
+func (cs *Coscheduling) PostFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod,
+	filteredNodeStatusMap framework.NodeToStatusMap) (*framework.PostFilterResult, *framework.Status) {
+	// check if failed due to preFilter failed, if not, filteredNodeStatusMap should not be nil
+	if filteredNodeStatusMap != nil {
```

Comment:
Nil filteredNodeStatusMap occurs only when PreFilter returns `Error`:

https://github.com/kubernetes/kubernetes/blob/v1.19.5/pkg/scheduler/core/generic_scheduler.go#L264-L266

In other words, non-nil filteredNodeStatusMap is the common case, right?

Reply:
We would like skip Prefilter Error, only react to Filter Error.
If Prefilter returns Unschedulable, filteredNodeStatusMap is not nil, so we skip.

Reply:
Thanks, will change, filteredNodeStatusMap must not be nil except some special cases